pyproject.toml

```diff
@@ -124,7 +124,10 @@ include = [
     { path = "rust/src/**", format = "sdist" },
 ]
 exclude = [
-    { path = "synapse/*.so", format = "sdist"}
+    { path = "synapse/*.so", format = "sdist" },
+    # This contains dev-only symlinks to the latest schema dumps. We don't want people
+    # to use it when installing.
+    { path = "synapse/storage/schema/latest_schema" },
```

Comment:
Would it make more sense to put this all in `contrib/datagrip` or something? That could contain the symlink and the README?

Reply:
That's a good idea!

Reply:
I think these changes can be backed out?